<div align="center">
<center>
	<h1>Marc A. Beer</h1>
	<h3>Genomics | Bioinformatics | Spatial analysis | Conservation</h3>
</center>
</div>


# About me
I am an interdisciplinary computational biologist interested in 1) applying and developing bioinformatic and spatial analysis tools for understanding population genomic variation, typically in a conservation biology context; and 2) testing the statistical performance of modern spatial population genomics methods using simulation. Thus far, most of my work falls under the subfields of population genomics referred to as landscape genetics and landscape genomics. Accordingly, I am experienced in various aspects of both spatial analysis and genomics analysis, such as the resolution of spatial patterns of gene flow and signatures of spatially divergent selection.

As a postdoctoral researcher, I am currently studying the genomic basis of adaptation to novel pathogens in wildlife. Adaptive evolution is one mechanism by which host populations may persist despite devastating population declines following disease emergence. A critical question concerns the extent of convergent evolution (broadly defined) among populations of the same host species; this question is both interesting from a fundamental perspective but also potentially important for conservation, as populations adapting to a shared selective pressure through different genomic mechanisms may warrant independent management. More broadly, we hope that greater understanding of rapid adaptation of host species can help clarify the role of evolution in population persistence in the face of emerging infectious diseases (EIDs), which are regarded as a leading threat to biodiversity.

My Ph.D. similarly focused on various facets of the biodiversity crisis. Specifically, I used population genomic data to study 1) the capacity of an imperiled endemic species (the streamside salamander, <i>Ambystoma barbouri</i>) to respond evolutionarily to threats posed by global climate change; 2) the evolutionary impacts of shifting ecological interactions between species (namely the evolution of the spotted-tailed quoll in response to population declines of its competitor, the Tasmanian devil); and 3) adaptive evolution of the invasive cane toad (<i>Rhinella marina</i>) across its Australian range. Accordingly, I have experience working in systems with clear conservation relevance (e.g., highly climate-sensitive species and invasive species), and I aim to build a career in applied ecology and evolutionary science to continue addressing conservation issues driven by rapid global change.

More information about me can be found on my <a href="https://marcabeer.github.io//">website</a>.

## Active interests
- Population genomics
- Spatial simulation
- Deep learning
- Genetic algorithms
- App development

# Tutorials & Workshops
- <a href="https://github.com/marcabeer/Nonrandom_missing_data_in_population_genetics">Identifying and fixing non-random missing data in population genetics (in progress)</a>
- <a href="https://github.com/marcabeer/Species_presence_simulation">Simulate species presence-[absence] data for ecological niche models</a>
- <a href="https://github.com/marcabeer/Simulate_regression_data">Simulate simple linear regression data</a>

# GitHub Stats
[![Marc's GitHub stats](https://github-readme-stats.vercel.app/api?username=marcabeer)](https://github.com/marcabeer/github-readme-stats)
